crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -121,10 +121,13 @@ impl TryFrom<&types::SetupMandateRouterData> for CybersourceZeroMandateRequest {
 
         let (payment_information, solution) = match item.request.payment_method_data.clone() {
             domain::PaymentMethodData::Card(ccard) => {
-                let card_issuer = ccard.get_card_issuer();
-                let card_type = match card_issuer {
-                    Ok(issuer) => Some(String::from(issuer)),
-                    Err(_) => None,
+                let card_type = match ccard
+                    .card_network
+                    .clone()
+                    .and_then(get_cybersource_card_type)
+                {
+                    Some(card_network) => Some(card_network.to_string()),
```

Comment:
Is this change compatible with cybersource docs?

Reply:
Yes, this change is compatible with cybersource docs, earlier we were passing `card_type`by converting  the card network derived from `get_card_issuer()` into cybs accepted value, but now we are converting   `card.card_network` if it is available, else use `get_card_issuer()`